### PR TITLE
feat: change landing page to be more generic

### PR DESCRIPTION
**Pull request by pratik151192**
Updated landing page to be more generic and list out 3 sentences for each service and link to their respective introduction/links. I also changed the heading to be more generic but I don't know if that affects SEO so please stop me if it does.

### CODE REVIEW

File: docs/intro.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 sidebar_position: 1
 sidebar_class_name: sidebar-item-overview
-sidebar_label: Momento Cache and Topics
-title: Momento Cache and Momento Topics Documentation
+sidebar_label: Momento
+title: Momento Documentation
 description: Learn what you need to know to speed up your cloud application with the world's first serverless cache.
```
Comment (pratik151192) [inline docs/intro.md:6]: I deliberately didn't change the description if it was better for SEO

File: docs/intro.md [hunk 2]
```diff
@@ -11,7 +11,12 @@ slug: /
...
+Momento provides instantly scalable and simply priced data platform you can take to production today. It currently provides
+the below suite of serverless distributed services:
+
+- Momento Cache is the world's first truly serverless caching service. It provides instant elasticity, scale-to-zero capability, zero infrastructure management, and blazing-fast performance. With Momento Cache, you grab the SDK, you get an end point, input a few lines into your code, and you're off and running. [Learn More](./introduction/what-is-serverless-caching.md)
+- Momento Topics is a high-speed, low-drag serverless pub/sub (publish/subscribe) messaging pattern that allows for real-time communication between parts of a distributed system. Momento Topics allows your apps to subscribe to topics to receive items published to the topic by your app’s publishers. [Learn More](./introduction/momento-topics.md)
+- Momento Vector Index simplifies vector database management with a truly serverless service, and provides the fastest path to get your AI project production ready at scale. [Join our waitlist](https://events.gomomento.com/momento-vector-index)
```
Comment (NoSQLKnowHow) [inline docs/intro.md:19]:
```suggestion
- Momento Vector Index (MVI) is a scalable, developer-friendly vector index service designed for real-time storage and retrieval of vector data for AI-powered applications. [Join our waitlist](https://events.gomomento.com/momento-vector-index)
```

File: docs/intro.md [hunk 3]
```diff
@@ -11,7 +11,12 @@ slug: /
   <meta name="google-site-verification" content="jXAJxglm5PRXRtL0R8kfGf80Xs1WM3IVAwAUPJC23lI" />
 </head>
 
-Momento Cache is the world's first truly serverless caching service. It provides instant elasticity, scale-to-zero capability, and blazing-fast performance. Gone are the days where you need to choose, manage, and provision capacity. With Momento Cache, you grab the SDK, you get an end point, input a few lines into your code, and you're off and running.
+Momento provides instantly scalable and simply priced data platform you can take to production today. It currently provides
+the below suite of serverless distributed services:
+
+- Momento Cache is the world's first truly serverless caching service. It provides instant elasticity, scale-to-zero capability, zero infrastructure management, and blazing-fast performance. With Momento Cache, you grab the SDK, you get an end point, input a few lines into your code, and you're off and running. [Learn More](./introduction/what-is-serverless-caching.md)
```
Comment (NoSQLKnowHow) [inline docs/intro.md:17]: While we are changing things, we should get the newer text for this. "world's first truly serverless caching..." need to be removed.

**Comment by NoSQLKnowHow** [comment]
With the new docs site split, this PR needs to be reworked with the new paths.

**Pull request closed without merging**